src/app/api/bollards.py

```diff
@@ -38,29 +38,51 @@ class BollardsValidationSchema(Schema):
 
     timeFrom = fields.Time(
         format="%H:%M",
-        required=True,
+        required=False,
     )
 
     timeTo = fields.Time(
         format="%H:%M",
-        required=True,
+        required=False,
     )
 
     @validates_schema
     def validate_dates(self, data, **kwargs):
+        if "timeTo" not in data or "timeFrom" not in data:
+            return
+
         if data["timeTo"] < data["timeFrom"]:
-            raise ValidationError("TimeTo must be later than timeFrom")
+            raise ValidationError("timeTo must be later than timeFrom")
+
+    @validates_schema
+    def validate_field_dependencies(self, data, **kwargs):
+        dependent_fields = {
+            "dayOfTheWeek": ["timeTo", "timeFrom"],
+            "timeFrom": ["dayOfTheWeek", "timeTo"],
+            "timeTo": ["dayOfTheWeek", "timeFrom"],
+        }
+
+        for field in dependent_fields:
```

Comment:
deze for loop kan eenvoudiger voelt het: 
ik lees 'alle velden moeten ingevuld zijn of allemaal niet' , 
Als niet alle velden ingevuld zijn -> error welke velden ontbreken
zoiets mischien? (let op:niet gechecked of werkt)
dependent_fields = ['dayoftheweek', 'timeTo', 'timeFrom']
if any(x in data for x in dependent_fields)

Reply:
Vraag is dus of je ook één of twee van de drie kan invullen? Weet jij dat?

Reply:
Dank voor de suggestie. Deze is zojuist doorgevoerd. Voor de drie velden `dayOfTheWeek`, `timeFrom` en `timeTo` is het inderdaad een situatie 'alle velden moeten ingevuld zijn of allemaal niet'.